mne/epochs.py

```diff
@@ -879,6 +879,10 @@ def drop_log_stats(self, ignore=['IGNORED']):
         -------
         perc : float
             Total percentage of epochs dropped.
+
+        See Also
+        --------
+        plot_drop_log
```

Comment:
for some of the functions, you have a brief blurb, but for some of the others, you don't.